### PR TITLE
Added emoji support in labels

### PR DESCRIPTION
**Pull request by ElkeBxl**
# Description
Labels didn't show emojis. They were changed by Github to for example : wrench :
I called the emojis endpoint Github provides ( https://developer.github.com/v3/emojis/ )

# Screenshots

# Issues
Solves no issues, is an enhancement

### CODE REVIEW

File: assets/js/app.js [hunk 1]
```diff
@@ -43,12 +43,16 @@ const app = new Vue({
...
 
-            fetch(`https://api.github.com/search/issues?page=${this.page}&q=language:${this.filterLanguage}+label:hacktoberfest+type:issue+state:open+${this.noReplyOnly && 'comments:0'}`)
-                .then(response => response.json())
-                .then(response => {
-
-                    let newResults = response.items.map(({repository_url, updated_at, ...rest}) => ({
-                      ...rest,
+			const issuesFetch = `https://api.github.com/search/issues?page=${this.page}&q=language:${this.filterLanguage}+label:hacktoberfest+type:issue+state:open+${this.noReplyOnly && 'comments:0'}`;
+			const emojisFetch = 'https://api.github.com/emojis';
+			Promise.all([issuesFetch, emojisFetch].map(url => fetch(url).then(response => response.json())))
```
[the remark this answers is not in the document]
Comment (ElkeBxl) [reply assets/js/app.js]: You mean chaining multiple fetches?

[the remark this answers is not in the document]
Comment (ElkeBxl) [reply assets/js/app.js]: Updated my code so they are chained, let me know if you find that better :)

[the remark this answers is not in the document]
Comment (damcclean) [reply assets/js/app.js]: yeah, that's better, thank you!

File: assets/js/app.js [hunk 2]
```diff
@@ -43,12 +43,19 @@ const app = new Vue({
         loadIssues() {
             this.isFetching = true;
 
-            fetch(`https://api.github.com/search/issues?page=${this.page}&q=language:${this.filterLanguage}+label:hacktoberfest+type:issue+state:open+${this.noReplyOnly && 'comments:0'}`)
-                .then(response => response.json())
-                .then(response => {
-
-                    let newResults = response.items.map(({repository_url, updated_at, ...rest}) => ({
-                      ...rest,
+			let emojis = [];
```
Comment (damcclean) [inline assets/js/app.js]: Could you not put this array in Vue's data, it can then be used as `then.emojies`?

Comment (ElkeBxl) [reply assets/js/app.js]: Done :)

**Comment by damcclean** [comment]
thanks for this! 😄

**Pull request merged**